cmake/std/atdm/ats1/custom_builds_unit_tests.sh

```diff
@@ -11,51 +11,26 @@ testAll() {
 
   ATDM_CONFIG_SYSTEM_DIR=${ATDM_CONFIG_SCRIPT_DIR}/ats1
 
-  ATDM_CONFIG_BUILD_NAME=before-intel-19.0.4-mpich-7.7.6_after
+  ATDM_CONFIG_BUILD_NAME=before-intel-19.0.4-mpich-7.7.15_after
   . ${ATDM_CONFIG_SCRIPT_DIR}/utils/set_build_options.sh
-  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.6 ${ATDM_CONFIG_COMPILER}
+  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.15 ${ATDM_CONFIG_COMPILER}
 
-  ATDM_CONFIG_BUILD_NAME=before-intel-19.0.4_mpich-7.7.6-after
+  ATDM_CONFIG_BUILD_NAME=before-intel-19.0.4_mpich-7.7.15-after
   . ${ATDM_CONFIG_SCRIPT_DIR}/utils/set_build_options.sh
-  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.6 ${ATDM_CONFIG_COMPILER}
+  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.15 ${ATDM_CONFIG_COMPILER}
 
   ATDM_CONFIG_BUILD_NAME=before_intel-19.0.4-after
   . ${ATDM_CONFIG_SCRIPT_DIR}/utils/set_build_options.sh
-  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.6 ${ATDM_CONFIG_COMPILER}
+  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.15 ${ATDM_CONFIG_COMPILER}
 
   ATDM_CONFIG_BUILD_NAME=before_intel-19-after
   . ${ATDM_CONFIG_SCRIPT_DIR}/utils/set_build_options.sh
-  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.6 ${ATDM_CONFIG_COMPILER}
+  ${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.15 ${ATDM_CONFIG_COMPILER}
 
```

Comment:
@e10harvey, try adding the unit test:

```
 ATDM_CONFIG_BUILD_NAME=intel
. ${ATDM_CONFIG_SCRIPT_DIR}/utils/set_build_options.sh
${_ASSERT_EQUALS_} INTEL-19.0.4_MPICH-7.7.15 ${ATDM_CONFIG_COMPILER}
```

It should crash and produce the error message.

Reply:
Thanks for adding the test with `ATDM_CONFIG_BUILD_NAME=intel`.